code/datums/beam.dm

```diff
@@ -165,6 +164,10 @@
 	anchored = TRUE
 	var/datum/beam/owner
 
+/obj/effect/ebeam/New(loc, beam_owner)
```

Comment:
`Initialize`?

Reply:
can probably change it over, but honestly i felt this var needs to be set before anything else happens ever cause things can quite literally explode if there's lag.

Reply:
Initialize is always ran right away in New